src/call/client.rs

```diff
@@ -427,16 +504,19 @@ struct ResponseStreamImpl<H, T> {
     read_done: bool,
     finished: bool,
     resp_de: DeserializeFn<T>,
+    headers_f: BatchFuture,
```

Comment:
When will it be used?

Reply:
Not sure I understand your comment. This is used here:
```
    pub async fn headers(&mut self) -> Result<Metadata> {
        Ok(Pin::new(&mut self.imp.headers_f).await?.initial_metadata)
    }
```